CodeEdit/Features/Settings/SettingsView.swift

```diff
@@ -84,6 +84,14 @@ struct SettingsView: View {
                 baseColor: .green,
                 icon: .system("externaldrive.fill")
             )
+        ),
+        // TODO: REMOVE THIS SECTION WHEN WE HAVE A SOLUTION FOR GETTING LSP BINARIES
+        .init(
+            SettingsPage(
+                .developer,
+                baseColor: .pink,
+                icon: .system("flask")
```

Comment:
The flask reminds me more of tests or experimental settings. Could we use something different? As a side note, this shouldn't block the PR, just something to keep in mind.

Reply:
Changed it to the bolt icon. Anyone have any ideas on a better icon?

Reply:
Looks good for now